schemas/isd.json

```diff
@@ -0,0 +1,236 @@
+         "description": "(v_x, v_y, v_z) velocity of sun in body-fixed frame.",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "items" : {
+               "type": "number",
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      },
+      "sensor_orientation" : {
+         "description": "Orientation of the sensor provided in quaternions",
+         "type": "array",
+         "items": {
+            "type": "array",
+            "description": "Each quaternion is of the form: (scalar, vector, vector, vector)" 
+            "items" : {
+               "type": "number",
+               "minItems": 4 ,
+               "maxItems": 4
+            }
+         }
+      },
+      "line_scan_rate": {
+         "description": "The line rates of the detector map with the associated start line and time",
+         "type": "array",
+         "items": {
+            "type": "array",
+             "items" : {
+               "type": "number",
+               "default: "-1",
+               "description": "The (start_line, line_start_time, exposure_time). The exposure time is
+                               in seconds and any non-provided values will be set to -1."
+               "minItems": 3,
+               "maxItems": 3
+            }
+         }
+      }, 
+      "detector_sample_summing": {
+         "description": "The summing in the sample-direction. A summing of 1 indicates no summing, 
+                         summing of 2 indicates 2 pixel summing, 4 indicates 4 pixel summing, etc",
+         "type": "number"
+      },
+      "detector_line_summing": {
+         "description": "The summing in the line-direction",
+         "type": "number"
+      },
+      "apply_light_time_correction": {
+         "description": "A flag to indcate whether the light-time correction should be applied",
+         "type": "bool"
+      },
+      "number_of_ephem": {
+         "description": "The number of emphemeris points",
+         "type": "number"
+      },
+      "number_of_quaternions": {
+         "description": "The number of quaternions",
+         "type": "number"
+      },
+      "dt_ephem": {
+         "description": "The time between each ephemeris point.",
+         "type": "number"
+      },
+      "t0_ephem": {
+         "description": "First ephemeris time - center image time",
+         "type": "number"
+      },
+      "dt_quat": {
+         "description": "Time spacing of quaternions.",
+         "type": "number"
+      },
+      "t0_quat": {
+         "description": "First quaternion time - center image time.",
+         "type": "number"
+      },
+      "reference_height": {
+         "description": "Used for determining the intersection of the look vector with the ellipsoid",
+         "type": "array",
+         "items":  {
+            "type": "number",
+            "minItems": 2,
+            "maxItems": 2
+         }
+      },
+      "interpolation_method": {
```

Comment:
This is another instance where having an enum of available types would be a nice to have.

Reply:
:+1: Agreed. I'm unsure what the available options are. Do you know what the list of things to include here would be?

Reply:
@twilson271828 Would be the person to go to on this.  I think the only method that is supported right now is lagrange

Reply:
Is it useful to switch to using an enumeration for this that currently only has one valid value?

Reply:
That sounds like a good idea to me